### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by MXueguang**
re https://github.com/castorini/pyserini/issues/353

### CODE REVIEW

File: docs/experiments-dpr.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Evaluate
 $ python scripts/dpr/evaluate.py --retrieval runs/run.nq-test.dpr.bf.bm25.json --topk 20
 $ python scripts/dpr/evaluate.py --retrieval runs/run.nq-test.dpr.bf.bm25.json --topk 100
 Top20  accuracy: 0.8260387811634349
-Top100 accuracy: 0.8700831024930747
+Top100 accuracy: 0.8858725761772853
```
Comment (ronakice) [inline docs/experiments-dpr.md:88]: I think you need to update this for the table too?

**Comment by MXueguang** [comment]
try to close https://github.com/castorini/pyserini/issues/347 by this PR too

**Review by lintool — APPROVED** [review]
lgtm. I might do another pass on the docs after.

**Review by ronakice — APPROVED** [review]
The numbers part look good to me, and since @lintool might do another pass later, we can merge!

**Pull request merged**